api/views.py

```diff
@@ -137,6 +138,25 @@ def get(self, request, project_id, stage_id):
             return paginator.get_paginated_response(data)
 
         return Response(status=status.HTTP_403_FORBIDDEN)
+    
+    @extend_schema(
+            request=TaskCreateSerializers,
+            responses=TaskCreateSerializers,
+    )
+    def post(self, request, project_id, stage_id):
+        project = get_object_or_404(Project, pk=project_id)
+        stage = get_object_or_404(Stage, pk=stage_id)
+        if is_pm_or_stage_owner(user=request.user, stage=stage, project=project):
+            request.data['stage'] = stage_id
+            serializer = TaskCreateSerializers(data=request.data)
+            if serializer.is_valid():
+                serializer.save()
+                
+                return Response(serializer.data, status=status.HTTP_201_CREATED)
+            
+            return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+        
+        return Response({'error': 'Không có quyền truy cập.'}, status=status.HTTP_403_FORBIDDEN) 
```

Comment:
em nên thay đổi cách thực thi check logic, nên check điều kiện false trước để raise lỗi vì điều kiện false thì khả năng xảy ra cao hơn nên ưu tiên check trc
vd:
```suggestion
        if not is_pm_or_stage_owner(user=request.user, stage=stage, project=project):
            return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
        ...
            
```